packages/in_app_purchase/in_app_purchase_storekit/shared/Classes/FIATransactionCache.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef NS_ENUM(NSUInteger, TransactionCacheKey) {
+  TransactionCacheKeyUpdatedDownloads,
+  TransactionCacheKeyUpdatedTransactions,
+  TransactionCacheKeyRemovedTransactions
+};
+
+@interface FIATransactionCache : NSObject
```

Comment:
I don't understand why this cache is necessary. Could you document its purpose above this interface declaration?

Reply:
This is just moved code; the scope of the review should be to enabling macOS support within the existing code, rather than changing/improving the code more generally.